doc/source/downloading_cesm.rst

```diff
@@ -19,22 +19,22 @@ software is at version 1.8.17. For more information or to download
 open source tools, visit `Subversion <http://subversion.apache.org/>`_
 and `git downloads <https://git-scm.com/downloads>`_.
 
-With valid git and svn clients installed on the machine where CESM will be
+With valid git and svn clients installed on the machine where CESM2 will be
 built and run, the user may download the latest version of the release
 code:
 
 .. code-block:: console
 
-    git clone https://github.com/ESCOMP/CESM.git my_cesm_sandbox
+    git clone -b release-cesm2.2.0 https://github.com/ESCOMP/CESM.git my_cesm_sandbox
```

Comment:
My original feeling was that this line should not be changed on master, since its intent is to give instructions for people who want to use the development code. That said: now that I think about it, I'm not sure if we want to have instructions out there that recommend using the latest version of master, since that could very well be something that hasn't gone through testing.

This leads me to think that maybe we should get rid of the built version from master ("Upcoming / beta version"), in which case it would be fine to bring in this line.

@fischer-ncar @mvertens what do you think?

Reply:
What if you introduced a branch named `latest_tested` and it points to the latest successfully-tested commit (for some definition of success)? Then developers could have an easy way to get to the latest / greatest development version without needing to constantly update this line in the documentation or risk getting an untested / broken set of externals.

This line in the documentation would be `git clone -b latest_tested https://github.com/ESCOMP/CESM.git my_cesm_sandbox` and @fischer-ncar would need to add a step when making a new tag: he would need to checkout `latest_tested` and run `git merge --ff-only [tested_tag]` (and push it back to github).

Reply:
@mnlevy1981 I like that idea. My gut feeling is that `latest_tested` would then just include betas, but we could discuss whether we'd want it to include alphas.

It seems like an entirely equivalent approach would be to make the `master` branch be this `latest_tested`, and introduce a new branch like `develop`... where whatever @fischer-ncar currently does to master he would now do to `develop`... correct me if I'm wrong @mnlevy1981 , but I think that ends up being the same as what you suggested, just with different names. I don't have strong feelings on which is better, but I kind of like the `master` / `develop` thing because it is a standard git convention (soon to become `main` / `develop`).

(However, I'll also acknowledge: I think this is effectively the idea I floated to @fischer-ncar and @mvertens a few weeks ago for the management of release branches, and at that time my idea was rejected as being unnecessarily complex for the value it added.)

Reply:
> It seems like an entirely equivalent approach would be to make the `master` branch be this `latest_tested`, and introduce a new branch like `develop`

It's very similar, but I think we want to keep `master` (or `develop`; I have no qualms about changing the name to be more descriptive) the default branch so that PRs are automatically pointed to that branch rather than `latest_tested`.

Reply:
I meant to leave that line alone, I've reverted it.  My thinking has always been, that `master` is untested, and use at your own risk.  I've always suggested people should be using a tag.  I have no problem with having a `master`/`cesm_development` work flow.  This is similar to what cam is doing.  However, a couple of weeks ago someone accidentally pushed an answer changing commit to the cam `master`.  So having a separate development branch isn't a 100% guarantee that master is correct.  Sounds like I just made an argument for @mnlevy1981 idea of a `latest_tested` branch.  Also another argument for having a `latest_tested` branch, as Mike mentioned, is the PRs.  Someone not closely coupled to CSEG, like Francis, isn't going to be sure where he needs to put his PR.  I've already ran into this issue with cime and cesm PRs.

Reply:
@fischer-ncar I don't understand all of your points relating to whether to use `master`/`latest_tested` or `develop`/`master` (or some other combination). It feels like it all comes down to what we would want the branches to be named and – the good point @mnlevy1981 points out – which branch to set as the default branch. (As for that latter point, I think there are arguments in both directions.)

But I think that this point is a very good one:

> I've always suggested people should be using a tag.

It does seem like, for the majority of cases where people are using CESM, they should be using a known, stable tag. So now I'm thinking that maybe the right thing to do here is to give instructions for how to find the available beta tags so that you can check out the one you want (something like `git tag --list '*beta*'`).

But I really don't have strong feelings about what we should do here, so if others do, I'm happy to go with that. Maybe that means we should just go with something like @mnlevy1981 's suggestion, since that seems to be the least change to the status quo while still providing some extra benefit... though I'd propose being more explicit and calling the new branch `latest_beta`.